lib/Models/WebProcessingServiceCatalogItem.js

```diff
@@ -155,14 +155,26 @@ WebProcessingServiceCatalogItem.prototype._load = function() {
         }
 
         for (var i = 0; i < outputs.length; ++i) {
-            if (!defined(outputs[i].Data) || !defined(outputs[i].Data.LiteralData)) {
+            if (!defined(outputs[i].Data)) {
                 continue;
             }
+            if (defined(outputs[i].Data.LiteralData)) {
+                that.shortReportSections.push(new ShortReportSection({
+                    name: outputs[i].Title,
+                    content: formatOutputValue(outputs[i].Title, outputs[i].Data.LiteralData)
+                }));
+            } else if (defined(outputs[i].Data.ComplexData)) {
+                var content = outputs[i].Data.ComplexData;
+                if (outputs[i].Identifier === "csv" || outputs[i].Identifier === "json")
```

Comment:
The WPS spec says `ComplexData` has a mandatory `Format` property, so that seems like the thing to use.